src/components/plots/ElbowPlot.jsx

```diff
@@ -130,9 +128,8 @@ const ElbowPlot = (props) => {
 };
 
 ElbowPlot.propTypes = {
-  experimentId: PropTypes.string.isRequired,
   config: PropTypes.object.isRequired,
-  plotData: PropTypes.array.isRequired,
```

Comment:
Why do we need to create `ElbowPlots` that do not get input `plotData`?

Reply:
They still get `plotData`, it is just no longer required.
I am following the pattern that has been taken in other plots of receiving an empty list by default.